code/hsec-tools/src/Security/OSV.hs

```diff
@@ -24,6 +24,11 @@ module Security.OSV
   , ReferenceType(..)
   , referenceTypes
   , Severity(..)
+
+  -- * /aeson/ re-exports
+  , encode
+  , eitherDecode
+  , eitherDecode'
```

Comment:
why do you reexport them?

Reply:
Just for convenience... if you're using `Security.OSV`, it's quite likely you'll be decoding from, or encoding to JSON.  Avoids an explicit import - and perhaps for some use cases it could avoid a direct dependency on *aeson*.

Reply:
I find it quite confusing, maybe it'll be clearer when it'll be in a library on its own.